### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by tjko** [tjko/jpegoptim#111] [Announcements]
* Parallel processing support added. Use option -w _max_ or --workers=_max_ to enable.
* Fix --stdin option
* Other minor fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/tjko/jpegoptim/releases/tag/v1.5.0'>v1.5.0</a>.</em>

**Comment by dirkmueller** [comment]
There is no gpg signature available anymore, is that intentional?

**Reply by tjko** [reply]
Thanks! I had forgotten to add those. They should now show up...

**Reply by dirkmueller** [reply]
looks good, thank you very much for the quick fix!
